airflow-core/src/airflow/api_fastapi/core_api/routes/public/assets.py

```diff
@@ -244,7 +244,7 @@ def get_asset_aliases(
     )
 
     return AssetAliasCollectionResponse(
-        asset_aliases=session.scalars(asset_aliases_select),
+        asset_aliases=list(session.scalars(asset_aliases_select)),
```

Comment:
Instead of having to copy here, would it be possible to just change the model declaration from `list` to `Iterable` instead?